docs/_news/news09.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: news
+title: New for Codewind 0.9.0
+description: New for Codewind 0.9.0
+keywords: news, new, updates, update, version, hybrid
+duration: 1 minute
+permalink: news09
+---
+
+## Codewind 0.9.0
+Tuesday, 18 February 2020
+
+#### ✨ New Features and Highlights for 0.9.0 ✨
+Introducing the tech preview of Codewind on the IntelliJ IDE! We want to make our tools available to as many developers as possible in their preferred IDEs. Many people have asked about IntelliJ, as it is a popular option among Java developers, so we are working hard to bring the effortless Codewind experience to IntelliJ. With this tech preview, you can now use Codewind on IntelliJ locally.
+
+![](images/imagesfornews/intelliJ-techpreview.gif){:width="800"}
+
+
+**IDEs (Both Eclipse and VS Code)**
+- Detailed application status notifications provide a [link to documentation if a problem is encountered](https://github.com/eclipse/codewind/issues/1812).
+- Depending on the level of support, you can [change the logging level in the IDE](https://github.com/eclipse/codewind/issues/1251). In Eclipse, first enable support features in the Codewind preferences and then right click on the connection in the Codewind Explorer view and select `Codewind Server Log Level`. In VS Code, you can access this configuration in the command palette - `Set Codewind Server Logging Level`.
+
+**VS Code**
+- If a remote connection disconnects due to network changes, such as the local machine going into sleep mode or switching networks, you can simply [refresh to reconnect](https://github.com/eclipse/codewind/issues/1776). 
+- There were styling updates to all webviews, including a better reaction to the theme. Instead of using grey icons for all themes, black and white icons are used and adjust to the theme. There were also fixes to the remote connection settings page - it should react better when the window is resized horizontally. 
+- Clicking on *Codewind View* or the Codewind View screenshot should open and focus the Codewind view.
```

Comment:
`Clicking on Codewind View...` is part of the welcome page, but the welcome page isn't mentioned here, so this point is lacking in context.

We should replace the above point with something like the following (and I think this should be the first point in the VS Code section):

- A Welcome Page has been added to help new users get started
  - From the Welcome Page, you can click on the **Codewind View** link or on the Codewind View screenshot to open the Codewind view.
  - Instructions and links are provided to make it easy to install Codewind on your local Docker Desktop, create connections to a remote Codewind instance, and create new and add existing projects.